src/System.Text.Primitives/System/Text/Parsing/PrimitiveParser2_bool.cs

```diff
@@ -10,6 +10,54 @@ public static partial class PrimitiveParser
     {
         #region helpers
 
+        private unsafe static bool IsTrue(byte* utf8Bytes, int length)
+        {
+            if (length < 4)
+                return false;
+
+            byte firstChar = utf8Bytes[0];
+            if (firstChar != 't' && firstChar != 'T')
```

Comment:
Please file an issue to possibly implement a case-sensitive fast parse that simply reads and int and compares it to a constant (as per the discussion we had in the previous commit version.